gradle-plugins/plugin-base/src/test/java/io/spine/gradle/GradleProject.java

```diff
@@ -145,11 +173,17 @@ public Builder setProjectFolder(TemporaryFolder folder) {
         }
 
         public Builder addProtoFile(String protoFileName) {
-            checkArgument(!isBlank(protoFileName));
+            checkArgument(!isNullOrEmpty(protoFileName));
             protoFileNames.add(protoFileName);
             return this;
         }
 
+        @SuppressWarnings("unused") // Part of API (used in other Spine modules)
+        public Builder addJavaFiles(String... fileNames) {
```

Comment:
As long as it was reported "unused" I assume there were no unit tests for this feature.

How do we ever know it works as expected?